source/manual/covid-19-services.html.md

```diff
@@ -164,9 +164,9 @@ settings and credentials for the backing service automatically.
 
 We have two AWS accounts (staging and production) for CDN and data
 storage. They contain DynamoDB, IAM and CloudFront resources,
-which were provisioned by Terraform in the [tech-ops-private
-repository](techops-repo). Members of the GOV.UK Coronavirus Services
-team have access.
+which were provisioned by Terraform in the [alphagov/covid-engineering
+repository][covid-engineering-repo]. Members of the GOV.UK Coronavirus
```

Comment:
sorry... I messed it up when editing and used squares, it kept formatting my comment as a link and it threw me, hence now the huge space! `[alphagov/covid-engineering
repository]  (techops-repo). Members of the GOV.UK Coronavirus Services	repository](covid-engineering-repo)`

Reply:
Weird! I think I fixed it now!